lib/vjs-hls.js

```diff
@@ -30,7 +31,49 @@ var attachVideojsStreamrootProvider = function (window, videojs, Hls) {
         }
 
         function _onError(event, data) {
-            console.error('HLS.js error: ' + data.type + ' - fatal: ' + data.fatal + ' - ' + data.details);
+            var error = {
+              message: ('HLS.js error: ' + data.type + ' - fatal: ' + data.fatal + ' - ' + data.details),
+            };
+            console.error(error.message);
```

Comment:
probably it might be better to only log the error if its fatal. Hls.js spits out a whole lot of errors on certain live streams with broken timestamps, but its not worth spaming the console with it. usually just scares people ;)